### DISCUSSION TITLE
I just want to ignore the line number

### DISCUSSION THREAD

**Discussion by CoffeeHwang** [mayurankv/Obsidian-Code-Styler#274] [Q&A] [answered]
hello.
I'm very grateful that it's a great feature and I'm using it.
However, it would be good to know how to ignore only the line number.
thank you very much.

**Comment by mayurankv** [comment]
Sorry, please could you elaborate? There are settings to turn off line numbers showing!

**Reply by CoffeeHwang** [reply]
Thank you for your interest so quickly.
To explain in detail, I want to set line numbers to `on` by default,
but for certain code blocks (for example, bash), there may be a way to `off` the line numbers by using the `ignore` keyword in the code block header or by going into the `Exclude Languages settings` and adding `bash`.
However, what I want is to exclude only line numbers for bash, and I do not want all the functions of the code styler (such as highlighting specific lines) to be ignored.

Just a simple suggestion, but I would like to exclude only line numbers using the following keyword (ignore-line-numbers), or if there is a better way.

```
    ```bash ignore-line-numbers
        sudo apt-get update
        sudo apt-get install docker
    ```
```

**Reply by br-mcc** [reply, marked as answer]
Perhaps you have already figured this out by now, but in case you haven't, you can turn off line number per block using `bash ln:false` in the block header. Is that what you were looking for? You can also turn off lines without a language specified by leaving a leading space ` ln:false`. I do this all the time for expressing terminal command output.  I.e. I'll have a bash block with highlighting+line numbers, and a separate output block of text with line numbers turned off.

You'll find the above stuff documented well in the plugin's page in Obsidian. Lots of other goodies in there, too!

**Reply by CoffeeHwang** [reply]
I was so happy that I got a solution that I forgot to give you feedback. It's late, but thank you.

**Reply by mayurankv** [reply]
So sorry I never got back to you! I must have completely missed this. Glad you got it sorted!
